Doc/c-api/exceptions.rst

```diff
@@ -53,8 +53,8 @@ Printing and clearing
 .. c:function:: void PyErr_PrintEx(int set_sys_last_vars)
 
    Print a standard traceback to ``sys.stderr`` and clear the error indicator.
-   **Unless** the error is a ``SystemExit``.  In that case the no traceback
-   is printed and Python process will exit with the error code specified by
+   **Unless** the error is a ``SystemExit``.  In that case no traceback is
```

Comment:
Do you think it would read better as: `**Unless** the error is a ``SystemExit``, in that case`?

The rest looks fine to me, thanks for improving the doc.

Reply:
@remilapeyre 
Yes, even better.  I assume you can amend/commit this yourself?